config.go

```diff
@@ -826,13 +828,15 @@ func buildTLSConfigForHttp2(config *Config) (*tls.Config, error) {
 	} else {
 		tlsCertPath := config.TLSCertPath
 		tlsKeyPath := config.TLSKeyPath
+		tlsExtraDomains := config.TLSExtraDomains
+		tlsExtraIps := config.TLSExtraIPs
```

Comment:
think we can just pass these straight in right? the reason the other's are short-handed here is so that we can do the `FileExist` checks